### DISCUSSION TITLE
Create UUID within cuefile ?

### DISCUSSION THREAD

**Discussion by subramanian0331** [cue-lang/cue#2164] [Q&A] [answered]
i want cue to create a random uuid everytime i run it.. is it possible ?

**Comment by myitcv** [comment, marked as answer]
```cue
exec cue eval -T x.cue

-- x.cue --
import "uuid"

i: int @tag(i, var=rand)
x: uuid.FromInt(i)
```

For more information please see `cue help injection`.

See also the notes a the top of https://pkg.go.dev/cuelang.org/go@v0.4.3/pkg/uuid regarding the `uuid` package.

**Reply by subramanian0331** [reply]
when i use it on command line it works fine... using cue eval.. but when i use go api i get following error
`incomplete value int (and 1 more errors)`

**Reply by mvdan** [reply]
@subramanian0331 you'll need to use `cue/load` with the default tags turned on; see https://pkg.go.dev/cuelang.org/go/cue/load#DefaultTagVars
